src/modules/Transcript/index.tsx

```diff
@@ -865,17 +905,18 @@ export interface TranscriptProps {
  * Renders Transcript
  */
 export const Transcript = (props: TranscriptProps): ReactElement => {
-    const { eventId: eventListEventId, onBack, initialSearchTerm, asrMode = false } = props;
+    const { eventId: eventListEventId, onBack, initialSearchTerm, useConfigOptions = false } = props;
+    const [eventId, setEventId] = useState(eventListEventId);
     const config = useConfig();
-    let eventId = eventListEventId;
+    const eventIdFromTicker = useLatestEventForTicker(config?.options?.ticker);
 
-    if (!eventId && config?.asrOptions?.eventId) {
-        eventId = config.asrOptions.eventId;
-    }
-
-    if (typeof eventId === 'undefined') {
-        return <div>No event found</div>;
-    }
+    useEffect(() => {
+        if (!eventId && config?.options?.eventId) {
+            setEventId(config.options.eventId);
+        } else if (!eventId && eventIdFromTicker.status === 'success') {
+            setEventId(eventIdFromTicker.state.data?.latestEventForTicker.id);
+        }
+    }, [setEventId, eventId, config, config.options, eventIdFromTicker.status, eventIdFromTicker]);
```

Comment:
Couldn't `config` be undefined here if you're always sending `config.options`?

Reply:
I dont think so, but I've added a ? to be safe